app/components/views/PrivacyPage/Privacy/Privacy.module.css

```diff
@@ -35,4 +35,20 @@
 }
 .contentTitleButtonsArea {
   margin-left: auto;
-}
\ No newline at end of file
+}
+.logs{
```

Comment:
Most of this css is already defined for the log panels in the Help tab. I guess we want the appearance for this new log panel to be the same, so should the existing css be reused rather than duplicating it here?